one-ui/src/main/java/com/marklogic/hub/curation/services/FlowManagerService.java

```diff
@@ -23,14 +25,21 @@
 import javax.ws.rs.NotFoundException;
 import java.io.IOException;
 import java.io.InputStream;
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
 
 @Service
 public class FlowManagerService {
```

Comment:
No action on this, but I'm realizing we have some naming inconsistency with Data Services, as the interface generated for DS is naturally going to be called a "Service". And then I was thinking that a Manager would be a layer on top that provides additional Java-based logic on top of one or more Services. But that conflicts with this usage of Service here, since it's the Service on top with Manager and other components (e.g. FlowRunner) below it. 

That said, I'm not sure how to resolve this. My initial thought is that perhaps the logic in here should move into FlowManager and FlowRunner. But again, no action needed for now, we can move this furniture around later.